pkg/cnservice/server.go

```diff
@@ -23,7 +31,15 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/pb/pipeline"
 )
 
-func NewService(cfg *Config) (Service, error) {
+var (
```

Comment:
remove to mo-service/version.go

Reply:
I will add a configuration item to transfer the MOVersion  from the mo-service to the cn-service away from the circular reference